index.html

```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
+    </h2>
```

Comment:
- [ ] Kindly 😃 try to use the correct font size for the heading to be like the Figma Design as I see you put the size of text here as 30 px, but in the Figma is 32px, so please try to fix that 😊
- [ ] Also please try to use the correct font weight to be like the Figma Design 😊
- [ ] Please try to decrease the margin-top of the heading here to be like the Figma design💯 

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/221970496-f24aa01a-e015-41b7-8404-483b044ae568.png)  |    ![image](https://user-images.githubusercontent.com/83360636/221970573-60a4efa7-1369-4559-bcb2-5eb265070446.png)  |